app/services/achiever/course/template.rb

```diff
@@ -76,6 +77,20 @@ def self.find_by_activity_code(activity_code)
     template
   end
 
+  # @return [Achiever::Course::Template|nil] nil if not found
+  def self.maybe_find_by_activity_code(activity_code)
+    templates = all
+    template = templates.find { |val| val.activity_code == activity_code.upcase }
+
+    return template if template
+
+    message = "Could not find template #{activity_code} in Smart Connector"
+    Rails.logger.error(message)
+    Sentry.capture_message(message, level: :error)
```

Comment:
```suggestion
    Sentry.capture_message(message, level: :warn)
```

Given that this should only cause some visual elements to not display, should we put this at warn level rather than error?

Reply:
I want this to be prominent in Sentry (and Slack us), because the immediate symptom is only visual, but it means a recommended course is actually missing and so the user can't book it.

Reply:
That is reasonable. It might be worth a comment explaining that reasoning or adding it as a note in your commit body